sample/src/main/res/layout/activity_playground_compose.xml

```diff
@@ -1,22 +1,21 @@
 <?xml version="1.0" encoding="utf-8"?>
-<androidx.constraintlayout.widget.ConstraintLayout
-    xmlns:android="http://schemas.android.com/apk/res/android"
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Keď som sto  skúšal lokálne tak som zistil že zmena typu animácií sa nijako neprejaví

Reply:
Bol to bug v compose beta03 kedy `animateAsState` nerešpektovalo nový animation spec, opravené v beta04: [https://android.googlesource.com/platform/frameworks/support/+/e03ed5a407b7717d2925fcc025b1d9e8ce91e476](https://android.googlesource.com/platform/frameworks/support/+/e03ed5a407b7717d2925fcc025b1d9e8ce91e476).